### PR TITLE
Move cloudfoundry tags with metadata to common metadata fields

### PR DESCRIPTION
**Pull request by jsoriano**
## What does this PR do?

In some Cloud Foundry deployments, metadata is included in tags, move these tags to the common fields.

## Why is it important?

Adding metadata with `add_cloudfoundry_metadata` implies querying the app metadata APIs, and storing the returned values in a local cache. This may be not needed in deployments that already include metadata the events from the firehose.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`. (https://github.com/elastic/beats/pull/22150#discussion_r512137845)

## Related issues

- Tags were added in #21177

### CODE REVIEW

**Comment by exekias** [comment]
Nice, any info on what versions/settings of CF include these? Should `add_cloudfoundry_metadata` skip events containing these fields as an optimization?

**Comment by jsoriano** [comment]
> any info on what versions/settings of CF include these?

Not sure about this, we know that these fields are included in new VMWare Tanzu Application Platform, but is not included in older PCF deployments.

> Should `add_cloudfoundry_metadata` skip events containing these fields as an optimization?

Yes, I will probably end up doing that. I was wondering if it would be better to explicitly use the processor where needed.

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -26,6 +26,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Update to Golang 1.12.1. {pull}11330[11330]
 - Disable Alibaba Cloud and Tencent Cloud metadata providers by default. {pull}13812[12812]
 - API address is a required setting in `add_cloudfoundry_metadata`. {pull}21759[21759]
+- Cloud Foundry tags including metadata are moved to common metadata fields. {pull}22150[22150]
```
Comment (jsoriano) [inline CHANGELOG.next.asciidoc:29]: Cloudfoundry tags will be added in 7.10.0. If we merge this change in time to include it in this release I will remove this changelog entry.

**Review by exekias — APPROVED** [review]

File: CHANGELOG.next.asciidoc (hunk 1 above)
Comment (jsoriano) [reply CHANGELOG.next.asciidoc]: Changelog entry removed, will re-add in a different PR if it doesn't reach 7.10.0.

**Pull request merged**